tests/src/readytorun/r2rdump/files/GcInfoTransitions/GcInfoTransitions.csproj

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I suggest using the .csproj's that you had before instead of making these netcoreapp projects. The majority of our test cases are not netcoreapp projects so we should try to keep these test cases consistent with the rest of the tests

Reply:
With the csproj I had before, the dll didn't have the generic method instances. For example, the xml didn't contain GenericFunction&lt;__Canon, Int32&gt;(Int32, __Canon)

I'm not sure why it's different when I use netcoreapp

Reply:
I'll go through and see if I can try and understand this... @fadimounir would you have any ideas here, why the previously written test cases (which are being removed in this PR) would be crossgen'ed with only one GenericFunction<_Canon, _Canon> instantiation?

Reply:
I pulled the latest CoreCLR master branch, and the GenericFunctions-test.xml (live-generated xml output by running the test) had the generic method instances, so from my end the previous tests look fine. I wonder what happened differently when you tried it.

Reply:
Could you check on this issue again before merging? I still find it surprising. But if we do need to make these all netcoreapp projects for the tests to be accurate then we can certainly make this change.